nexus/db-model/src/schema_versions.rs

```diff
@@ -28,6 +28,8 @@ static KNOWN_VERSIONS: LazyLock<Vec<KnownVersion>> = LazyLock::new(|| {
         // |  leaving the first copy as an example for the next person.
         // v
         // KnownVersion::new(next_int, "unique-dirname-with-the-sql-files"),
+        KnownVersion::new(203, "scim-users-and-groups-lower"),
+        KnownVersion::new(202, "scim-actor-audit-log"),
```

Comment:
Oops I didn't realize this PR already touched this and can probably collapse it into one version bump. Will fix this soonish when back at the keyboard.